pkg/drivers/libvirt/libvirt_nonlinux.go

```diff
@@ -0,0 +1,12 @@
+// +build !linux
+
+package libvirt
+
+import (
+	"github.com/code-ready/machine/drivers/errdriver"
+	"github.com/code-ready/machine/libmachine/drivers"
+)
+
+func NewDriver(hostName, storePath string) drivers.Driver {
+	return &errdriver.Driver{}
```

Comment:
@guillaumerose is it there just because if someone use `libvirt.NewDriver` in case of mac/windows it will return the error?

Reply:
yes it was a bad idea. It was only for the linter to be able to compile. I removed it.